src/backend/EdFi.DataManagementService.Backend.Postgresql.Test.Integration/GetTests.cs

```diff
@@ -67,16 +65,8 @@ public async Task Setup()
         public void It_should_be_found_by_get_by_id()
         {
             _getResult!.Should().BeOfType<GetResult.GetSuccess>();
-            var successResult = _getResult as GetResult.GetSuccess;
-            successResult!.DocumentUuid.Value.Should().Be(_documentUuidGuid);
-            
-            var actualJson = JObject.Parse(successResult!.EdfiDoc.ToJsonString());
-
-            var expectedJson = JObject.Parse(_edFiDocString);
-            expectedJson["id"] = _documentUuidGuid;
-
-            actualJson.Should()
-                .BeEquivalentTo(expectedJson, options => options.ComparingByMembers<JObject>());
+            (_getResult! as GetResult.GetSuccess)!.DocumentUuid.Value.Should().Be(_documentUuidGuid);
+            (_getResult! as GetResult.GetSuccess)!.EdfiDoc.ToJsonString().Should().Contain("\"abc\":1");
```

Comment:
Simplified these, as "should contain" should be all the info we need.

Reply:
And prefer to avoid Newtonsoft

Reply:
Agree. And the E2E tests fully cover the JSON documents.